### DISCUSSION TITLE
Exceeded maximum number of files in tree

### DISCUSSION THREAD

**Discussion by aloeenmae** [jfrog/jfrog-cli#2716] [Q&A] [answered]
Hi!

Can anyone explain what this "**Exceeded maximum number of files in tree**" exactly means?
```
JFROG_CLI_LOG_LEVEL=DEBUG jf rt upload --url https://my-artifactory/artifactory --insecure-tls --user XXXXX --password XXXX --threads 10 "*" my-docker/
12:14:25 [Debug] JFrog CLI version: 2.70.0
12:14:25 [Debug] OS/Arch: linux/amd64
....
12:15:13 [Debug] Sending HTTP PUT request to: https://my-artifactory/artifactory/my-docker/some_image/v1.0/sha256__xxxxxxxxxxxxxxxxxx
12:15:14 [Debug] [Thread 5] Artifactory response: 201   (Checksum deploy)
12:15:14 [Debug] Uploaded 1305 artifacts.
12:15:14 [🔵Info] Exceeded maximum number of files in tree
```
I see this "**Exceeded maximum number of files in tree**" 1103 times in my log files.

**Comment by Or-Geva** [comment, marked as answer]
Hey @aloeenmae, thanks for pointing that out! We've put up a [PR](https://github.com/jfrog/jfrog-cli-core/pull/1279) to fix the 1103 times log issue. Appreciate the heads-up!.
In general, the JFrog CLI usually shows a visual tree of what’s been uploaded once it's done. But if there’s too much stuff, it won’t show anything to keep the terminal clean.

**Comment by aloeenmae** [comment]
Thanks!
